modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -0,0 +1,109 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-command-reference_{context}"]
+= Pipelines as Code command reference 
+
+[role="_abstract"]
+The `tkn-pac` CLI tool offers the following capabilities:
+
+* Bootstrap Pipelines as Code installation and configuration
+* Create a new Pipelines as Code repository
+* List all Pipelines as Code repositories
+* Describe a Pipelines as Code repository and the associated runs
+* Generate a simple pipeline run to get started
+* Resolve a pipeline run, as if it was executed by Pipelines as Code on service
+
+== Basic syntax
+
+[source,terminal]
+----
+$ tkn pac [command or options] [arguments]
+----
+
+== Global options
+
+[source,terminal]
+----
+$ tkn pac --help
+----
+
+== Utility commands
+
+=== bootstrap
+
+.Bootstrapping Pipelines as Code installation and configuration
+[options="header"]
+|===
+
+| Command | Description 
+
+| `tkn pac bootstrap` | Installs and configures Pipelines as Code for Git repository hosting service providers, such as GitHub and GitHub Enterprise. 
+
+| `tkn pac bootstrap --nightly` | Installs the nightly build of Pipelines as Code.
+
+| `tkn pac bootstrap --route-url <public_url_to_ingress_spec>` | Overrides the OpenShift route URL. 
+
+By default, `tkn pac bootstrap` detects the OpenShift route which is automatically associated with the Pipelines as Code controller service. 
+
+If you do not have a {product-title} cluster, it asks you for the public URL that points to the ingress specification.
```

Comment:
```suggestion
If you do not have a {product-title} cluster, it asks you for the public URL that points to the ingress endpoint.
```
Should this be `endpoint` rather than `specification`?